src/common/PRIMARY_STATS.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
Did you consider that according to feasts "stamina" is a primary stat for tanks?

Reply:
I did consider it, but I wasn't sure of a solution. I thought for other things tanks had their damage stat as primary?

Reply:
There's no actual bug yet because the new field is only hooked into EoE.

Reply:
Ok my main argument for doing it this way is that it's easy to convert to stam for a tank:
```
if (spec.role === ROLES.TANK) {
  return 'Stamina';
} else {
  return spec.primaryStat;
}
```
But if I made STAMINA primary stat for all tank specs, there'd be no way to get their 'damage primary'.